docs/Home.md

```diff
@@ -41,7 +41,8 @@ to publish deployments built within Jenkins to your Kubernetes clusters running
     export SA_EMAIL=$SA@$PROJECT.iam.gserviceaccount.com
     gcloud projects add-iam-policy-binding --member serviceAccount:$SA_EMAIL --role roles/iam.serviceAccountUser $PROJECT
     gcloud projects add-iam-policy-binding --member serviceAccount:$SA_EMAIL --role roles/container.clusterAdmin $PROJECT
-    gcloud projects add-iam-policy-binding --member serviceAccount:$SA_EMAIL --role roles/container.admin $PROJECT
+    gcloud projects add-iam-policy-binding --member serviceAccount:$SA_EMAIL --role roles/container.developer $PROJECT
```

Comment:
container.viewer should be the only required role now.

Reply:
only required "container" service role I mean.

Reply:
Just tried it out. container.developer is needed for "container.deployments.create" permissions.

Reply:
Makes sense. Thanks for following up on it.

Reply:
Took another look at the article linked in the issue. Would prefer to model its approach and use a locked down set of permissions in GKE and specify granular permissions in RBAC. Take another look and let me know if you have any questions: https://codeascraft.com/2018/06/05/deploying-to-google-kubernetes-engine/

Reply:
One thing which isn't clear from the article, is that when you use token based auth, the user in the rolebinding needs to be the service accounts ID not email. You can get that using this command:
gcloud iam service-accounts describe $SA_EMAIL --format 'value(oauth2ClientId)'

Reply:
Additionally, the RBAC role definition itself will need to include "apps" and "extensions" in apiGroups.

Reply:
Interesting. Will we want to include that in this PR?

Reply:
Yes, the point of this PR is to enable a more secure permissions set for our customers.

Reply:
I found a cleaner way regarding the user for the RBAC rolebinding. If you add the following as a scope requirement list when you retrieve the credential, the role binding will recognize the SA's email: "https://www.googleapis.com/auth/userinfo.email" 
This alleviates the need to use the OAuth client id.

Reply:
that's a great find. Thanks!